pkg/cli/admin/upgrade/status/status.go

```diff
@@ -260,6 +279,21 @@ func (o *options) Run(ctx context.Context) error {
 		fmt.Fprintf(o.ErrOut, "warning: No current %s info, see `oc describe clusterversion` for more details.\n", clusterStatusFailing)
 	}
 
+	// get the alerts for the cluster. if we're unable to fetch the alerts, we'll let the user know that alerts
+	// are not being fetched, but rest of the command should work.
+	var alertData AlertData
+	alertBytes, err := o.getAlerts(ctx)
+	if err != nil {
+		fmt.Println("Unable to fetch alerts from thanos, ignoring alerts in 'Update Health': ", err)
```

Comment:
nit: "thanos" -> "Thanos", to match the casing preferred by the Thanos folks, for [example][1]:

> Join users and companies that are using Thanos in production.

[1]: https://thanos.io/

Reply:
although actually, this call-site has no idea that we're reaching out to Thanos; that's an implementation detail.  I'd expect `inspectalerts` to be providing sufficient context for debugging failed requests if we fail to fetch alerts, and here we can probably just do:

```golang
fmt.Fprintf(o.ErrOut, "warning: Unable to fetch alerts, ignoring alerts in 'Update Health': %v\n", err)
```

which will also get us logging this warning to `ErrOut`, as we already do [here][1].

[1]: https://github.com/openshift/oc/blob/5ffe8230ec8aa94dc37dfa589c108402d12f8615/pkg/cli/admin/upgrade/status/status.go#L260